inspirehep/submissions/views.py

```diff
@@ -156,6 +165,158 @@ def start_workflow_for_submission(self, submission_data, control_number=None):
+        # (after de-serialization if it's missing from input data)
+        # this fields will be removed from record
+        additional_fields = [
+            "external_job_identifier",
+            "accelerator_experiments",
+            "urls",
+            "contact_details",
+            "reference_letters",
+        ]
+
+        if not is_superuser_or_cataloger_logged_in():
+            old_status = record.get("status", "pending")
+            new_status = data.get("status", old_status)
+            if new_status not in self.user_allowed_status_changes[old_status]:
+                raise RESTDataError(
+                    f"Only curator can change status from '{old_status}' to '{new_status}'."
+                )
+        record_data = dict(record)
+        for key in additional_fields:
+            if key not in data and key in record_data:
+                del record_data[key]
+        record_data.update(data)
+        builder = JobBuilder(record=record_data)
+        return builder
+
+    def prepare_data(self, data, record=None):
+        """Prepares data received from form.
+
+        As jobs do not have any 'workflows' it's required to set all the logic
+        for updating record from data provided by the user somewhere..."""
+
+        if record:
+            builder = self.prepare_update_record(data, record)
+        else:
+            builder = self.prepare_new_record(data)
+
+        try:
+            builder.validate_record()
+        except ValidationError as e:
+            logger.error(f"Cannot process job submission: {e}")
+            raise RESTDataError(e.args[0])
+        except SchemaError as e:
+            logger.error(f"Schema is broken")
+            abort(500, str(e))
+        data = builder.record
+        return data
+
+    def user_can_edit(self, record):
+        if is_superuser_or_cataloger_logged_in():
+            return True
+        acquisition_source = record.get("acquisition_source")
+        if (
+            acquisition_source.get("orcid") == self.get_user_orcid()
+            and acquisition_source.get("email") == current_user.email
+            and record.get("status") != "closed"
+        ):
+            return True
+        return False
+
+    def create_ticket(self, record, rt_template):
+        rt_queue = "JOBS"
+        requestor = record["acquisition_source"]["email"] or record[
+            "acquisition_source"
+        ].get("name", "UNKNOWN")
+        control_number = record["control_number"]
+        rt_template_context = {
+            "job_url": current_app.config["JOB_DETAILS"].replace(
+                "<pid_value>", str(control_number)
+            ),
+            "job_url_edit": current_app.config["JOB_EDIT"].replace(
+                "<pid_value>", str(control_number)
+            ),
+            "hep_url": current_app.config["INSPIREHEP_URL"],
+        }
+        try:
+            _ = create_ticket_with_template(
```

Comment:
``create_ticket_with_template`` returns ticket ``id`` if it's ``-1`` then there was something wrong we should log this